app/models/partnership.rb

```diff
@@ -1,6 +1,21 @@
 # frozen_string_literal: true
 
 class Partnership < ApplicationRecord
+  enum status: {
+    pending: "pending",
+    accepted: "accepted",
+    rejected: "rejected",
+  }
```

Comment:
Don't think we have a "pending"

Reply:
So what would be the status before the school has confirmed the partnership then, nil?

Reply:
Schools don't confirm partnerships in the [current designs](https://miro.com/app/board/o9J_leK0NUo=/?moveToWidget=3074457355028269710&cot=14)